api/odigos/v1alpha1/instrumentedapplication_types.go

```diff
@@ -51,23 +51,6 @@ const (
 	ProcessingStateSkipped   ProcessingState = "Skipped"
 )
 
-// +kubebuilder:object:generate=true
-type RuntimeDetailsByContainer struct {
-	ContainerName  string                     `json:"containerName"`
-	Language       common.ProgrammingLanguage `json:"language"`
-	RuntimeVersion string                     `json:"runtimeVersion,omitempty"`
-	EnvVars        []EnvVar                   `json:"envVars,omitempty"`
-	OtherAgent     *OtherAgent                `json:"otherAgent,omitempty"`
```

Comment:
`OtherAgent` and `EnvVar` are also declared in this file. should we also move them to `instrumentationconfig` along with `RuntimeDetailsByContainer`?